pkg/utils/codegeneration/utils.go

```diff
@@ -0,0 +1,39 @@
+package codegeneration
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"golang.org/x/tools/imports"
+)
+
+func PrintFiles(dir string, files map[string]string) error {
+	for file, content := range files {
+		if !strings.HasPrefix(file, dir) {
+			file = dir + "/" + file
+		}
+
+		if strings.HasSuffix(file, ".go") {
+			imports.LocalPrefix = "github.com/smartcontractkit"
+			rawContent, err := imports.Process(file, []byte(content), nil)
+			if err != nil {
+				// print an error, but also write the file so debugging the generator isn't a pain.
+				fmt.Printf("Error formatting file %s: %s\n", file, err)
```

Comment:
Always writing makes sense, but is just printing the error sufficient? Or should it also be returned?

Reply:
I didn't want to return errors there because it might prevent other files from being written.

Reply:
fair, I was thinking that the generator probably shouldn't stop other work, but this is the last step anyways.  Also, callers can do what they want with the errors anyway.